### PR TITLE
chore: update minikube installation to helm v3

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

Following the official installation step on [release installation suggestion](https://github.com/helm/helm/releases/tag/v3.4.1) and [doc installation steps](https://helm.sh/docs/intro/install/#from-script), update helm installation from v2 to v3.

### CODE REVIEW

File: docs/get_started/get_started_on_minikube.md [hunk 1]
```diff
@@ -28,14 +28,9 @@ Perform the following steps to set up the local Kubernetes environment:
 2. Install helm:
```
Comment (WangXiangUSTC) [inline docs/get_started/get_started_on_minikube.md:28]: how about adding a link https://helm.sh/docs/intro/install/#from-script

**Review by WangXiangUSTC — COMMENTED** [review]
rest LGTM

File: docs/get_started/get_started_on_minikube.md (hunk 1 above)
Comment (Yiyiyimu) [reply docs/get_started/get_started_on_minikube.md:28]: Fixed. Since "from script" is not necessary, I just paste https://helm.sh/docs/intro/install

**Review by WangXiangUSTC — COMMENTED** [review]
LGTM

**Review by g1eny0ung — APPROVED** [review]

**Pull request merged**